crates/swc_ecma_parser/src/lexer/util.rs

```diff
@@ -174,6 +186,14 @@ impl<'a> Lexer<'a> {
     /// See https://tc39.github.io/ecma262/#sec-white-space
     pub(super) fn skip_space(&mut self, lex_comments: bool) -> LexResult<()> {
         loop {
+            let skip = SkipWhitespace::new(false);
+            let skip = skip.simd(self.input.as_str().as_bytes());
+            // dbg!(skip.offset);
+            for _ in 0..skip.offset {
+                self.input.bump();
+            }
```

Comment:
need to bump all at once for the speed improvement.

Reply:
Yeah, I'll do it too. Thank you!